boards/arm/stm32h747i_disco/stm32h747i_disco.dtsi

```diff
@@ -64,6 +64,24 @@
 			status = "disabled";
 		};
 	};
+
+	dsi_lcd_qsh_030: connector_dsi_lcd {
+		compatible = "st,dsi-lcd-qsh-030";
```

Comment:
You need to define this binding in `dts/bindings/gpio`.
Otherwise it won't be taken into account

Reply:
sorry, I missed that.